deployment/hasura/metadata/databases/tables/sequencing/user_sequence.yaml

```diff
@@ -26,24 +29,24 @@ select_permissions:
 insert_permissions:
   - role: aerie_admin
     permission:
-      columns: [definition, seq_json, name, parcel_id]
+      columns: [definition, seq_json, name, parcel_id, workspace_id]
       check: {}
       set:
         owner: "x-hasura-user-id"
   - role: user
     permission:
-      columns: [definition,seq_json, name, parcel_id]
+      columns: [definition, seq_json, name, parcel_id, workspace_id]
       check: {}
       set:
         owner: "x-hasura-user-id"
 update_permissions:
   - role: aerie_admin
     permission:
-      columns: [definition,seq_json, name, owner, parcel_id]
+      columns: [definition, seq_json, name, parcel_id, workspace_id]
       filter: {}
   - role: user
     permission:
-      columns: [definition,seq_json, name, owner, parcel_id]
+      columns: [definition, seq_json, name, parcel_id, workspace_id]
```

Comment:
`Owner` was also dropped here. Since only the `user` who owns this `user_sequence` can update the row, they should be allowed to change the ownership to someone else.